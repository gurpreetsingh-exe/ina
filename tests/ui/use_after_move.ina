// COMPILE_FAIL

type Test = {}

fn main() {
    let a = Test {};
    let b = a;
    let c = a;
}

