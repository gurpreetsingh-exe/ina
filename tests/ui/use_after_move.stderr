error: use of moved value `a`
   ╭─[tests/ui/use_after_move.ina:8:13]
   │
 7 │     let b = a;
   │             ┬
   │             ╰─ value moved here
   ∙
 8 │     let c = a;
   │             ┬
   │             ╰─ value used here after move
   ∙
───╯
